src/Notifications/NotificationsModel.cs

```diff
@@ -61,5 +61,8 @@ internal class NotificationItemModel
 
         [DataMember(Name = "isPinned")]
         internal bool IsPinned { get; set; }
+
+        [DataMember(Name = "isUnread")]
+        internal bool IsUnread { get; set; } = false;
```

Comment:
Usually we would name bool not using `Un`, e.g. `IsPinned` `IsHidden` `IsVisible`, I would suggest to use `IsRead`

Reply:
Right, It's changed in the notifications center too. Thanks!